### PR TITLE
Adding content about the Recommended products preview panel

### PR DESCRIPTION
**Pull request by erikmarr**
## Purpose of this pull request

This pull request (PR) adds content about the new Recommended products preview panel used when creating a new recommendation unit. Also updating some text in the the page builder content.

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/marketing/create-new-rec.html
- https://docs.magento.com/user-guide/marketing/page-builder-add-product-recs.html

### CODE REVIEW

File: src/marketing/create-new-rec.md [hunk 1]
```diff
@@ -66,3 +66,20 @@ After you create and activate the recommendation unit, Magento will begin [colle
...
+
+## Preview Recommendations {#preview}
+
+Use the **Recommended products preview** panel to preview what products will be displayed in a recommendation unit when deployed to your storefront.
+
+![Recommendation preview]({% link images/images-ee/prex-preview.png %}){: .zoom}
+
+|Field|Description|
+|---|---|
+|**Name**|Name of the product|
```
Comment (leslietilling) [inline src/marketing/create-new-rec.md:78]: We don't usually bold the terms in a field description list. Doesn't really need it.

**Review by leslietilling — APPROVED** [review]
Just a minor formatting inconsistency

File: src/marketing/create-new-rec.md [hunk 2]
```diff
@@ -66,3 +66,22 @@ After you create and activate the recommendation unit, Magento will begin [colle
...
+
+Use the **Recommended products preview** panel to preview what products will be displayed in a recommendation unit when deployed to your storefront.
+
+![Recommendation preview]({% link images/images-ee/prex-preview.png %}){: .zoom}
+
+|Field|Description|
+|---|---|
+|Name|Name of the product|
+|SKU|Stock Keeping Unit of the product|
+|Price|Cost excluding taxes and discounts of the product|
```
Comment (misha-kotov) [inline src/marketing/create-new-rec.md:80]:
```suggestion
|Price|Price of the product|
```

File: src/marketing/create-new-rec.md [hunk 3]
```diff
@@ -66,3 +66,22 @@ After you create and activate the recommendation unit, Magento will begin [colle
...
+|Result Type|**Primary** indicates that there is enough training data collected to display a high quality recommendation. **Backup** indicates there is not enough training data collected so a backup recommendation is used to fill the slot. See [Time and training for machine learning models]({% link marketing/product-recommendations.md %}#trainmlmodels) to learn more about machine learning models and backup recommendations|
```
Comment (misha-kotov) [inline src/marketing/create-new-rec.md:81]:
```suggestion
|Result Type|**Primary** indicates that there is enough training data collected to display a recommendation. **Backup** indicates there is not enough training data collected so a backup recommendation is used to fill the slot. See [Training time for machine learning models]({% link marketing/product-recommendations.md %}#trainmlmodels) to learn more about machine learning models and backup recommendations|
```

File: src/marketing/recommendation-incl-excl.md [hunk 4]
```diff
@@ -10,7 +10,11 @@ Conditions can be static or dynamic. A static condition uses existing product at
 
 {:.bs-callout-info}
 Inclusion and exclusion filters replace the legacy category exclusions in versions 3.2.2 and later of the `magento/product-recommendations` module. See the [release notes](https://devdocs.magento.com/recommendations/release-notes.html) to learn more.
+### Default filters {#default}
 
+Magento automatically applies non-configurable default filters to recommendation units.
+
+If you have multiple recommendation units deployed to a product detail page, Magento filters out duplicated products. For example, if a product is returned in the third position for the second recommendation unit, but it is already displayed in the first recommendation unit on the page, Magento does not display that product in the second recommendation unit. On the cart page, Magento filters out products that you have previously purchased and products currently in your cart.
```
Comment (misha-kotov) [inline src/marketing/recommendation-incl-excl.md:17]:
```suggestion
If you have multiple recommendation units deployed to a page, Magento filters out product that would be repeated among these units. The first recommendation unit on a page is not affected by product deduplication. For every other recommendation unit, products already shown in previous units are filtered out. That way you don't waste your merchandising space on items already displayed to the shopper. 
Additionally, Magento filters out previously purchased products and those currently in the cart.
```

**Review by misha-kotov — APPROVED** [review]
Looks good

**Comment by dshevtsov** [comment]
The work on this pull request will continue internally at magento-commerce/merchdocs#1.

**Pull request closed without merging**